Sources/Carting/Commands/Info.swift

```diff
@@ -13,8 +13,11 @@ struct Info: ParsableCommand {
     @Option(name: [.short, .long], default: ProcessInfo.processInfo.environment["PROJECT_DIR"], help: "The project directory path.")
     var path: String?
 
+    @Argument(help: "The project directories that contains frameworks to proceed")
+    var frameworksDirectoryPaths: [String]
+
```

Comment:
If it is a mandatory argument, it would be better to use some default values here. What do you think?

Reply:
[String] will be simplified to String. Running info for multiple directories doesn't seem to be useful